src/core/Microsoft.PowerToys.Settings.UI/Controls/HotkeySettingsControl.xaml.cs

```diff
@@ -40,14 +40,16 @@ public bool Enabled
                 if (value)
                 {
                     HotkeyTextBox.IsEnabled = true;
-                    //TitleText.IsActive = "True";
-                    //TitleGlyph.IsActive = "True";
+

```

Comment:
@crutkas any reason why we wouldn't just delete the comments?

Reply:
I wasn't the one who commented them so going to let them live for a bit longer :)